contracts/token/IERC677.cairo

```diff
@@ -0,0 +1,41 @@
+%lang starknet
+
+from starkware.cairo.common.uint256 import Uint256
+
```

Comment:
The ERC677 interface alone. We're defining it here, but can't inherit anywhere as far as I can tell 🤔